src/components/home/header.svelte

```diff
@@ -1,21 +1,26 @@
 <script>
+  import { stores } from '@sapper/app';
   import Button from 'ui/button.svelte';
-  // import NotificationCenter from '@/components/profile/notification-center.svelte';
-  import { isAuthed, login, user } from '@/store/user';
+  import NotificationCenter from '@/components/common/notification-center.svelte';
+  import { login, loginCheat } from '@/utils/auth.js';
+  import { ENV } from '@/constants/env.js';
+
+  const loginFunction = (ENV === 'development' ? loginCheat : login);
```

Comment:
Repetition (Here and in main header). Choosing the *loginFunction* could be performed explicitly in *auth.js*